### ISSUE TITLE
Complete clear cache

### ISSUE THREAD

**Issue by atomixstar** [nexcess/magento-turpentine#1380]
Hello, nice extension. I use this with Magento 1.9 - apache2 - varnish 3 on AWS EC2.

On first install was some errors, I fixed them and now I want to complete clear cache for all pages in my domain. I tried everything but not working, Clear cache via Magento admin, clear via ssh nothing, the cache still active. Header cart not updating after add to cart with ajax extension but I found some solutions here about that. I added to turpentine_esi.xml restarted varnish via ssh but still not nothing, cache is not clearing, on some page cart header appears 4 items on other appear 1-2 etc.

Also, is there any option to disable temporary cache on varnish during development?

Thank you

**Comment by shine-eglobeits** [comment]
yes, you can bypass by adding a cookie. sorry I forgot that cookie name will update you once I get.

**Comment by miguelbalparda** [comment]
[Here](https://github.com/nexcess/magento-turpentine/wiki/Usage) is the Wiki section explaining how to use the Activate Varnish Bypass feature.

**Comment by miguelbalparda** [comment]
[Here](https://github.com/nexcess/magento-turpentine/wiki/Usage) is the Wiki section explaining how to use the Activate Varnish Bypass feature.

**Comment by miguelbalparda** [comment]
[Here](https://github.com/nexcess/magento-turpentine/wiki/Usage) is the Wiki section explaining how to use the Activate Varnish Bypass feature. Also, the issue you are experiencing is not related to Varnish but to an incompatible theme architecture. When you see different items in your cart, you are basically seeing others people carts because the block is being cached publicly instead of private. i suggest you to check the rest of the Wiki where you can find examples of how to set up an ESI policy.
